src/main/java/uk/gov/hmcts/befta/util/MapPlaceholders.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.befta.util;
+
+public enum  MapPlaceholders {
+
+    ANYTHING_PRESENT,
+    ANYTHING_IF_EXISTS,
+
+    ANY_NULLABLE,
+    ANY_STRING_NULLABLE,
+    ANY_NUMBER_NULLABLE,
+    ANY_INTEGER_NULLABLE,
+    ANY_FLOATING_NULLABLE,
+    ANY_DATE_NULLABLE,
+    ANY_TIMESTAMP_NULLABLE,
+    ANY_OBJECT_NULLABLE,
+
+    ANY_NOT_NULLABLE,
+    ANY_STRING_NOT_NULLABLE,
+    ANY_NUMBER_NOT_NULLABLE,
+    ANY_INTEGER_NOT_NULLABLE,
+    ANY_FLOATING_NOT_NULLABLE,
+    ANY_DATE_NOT_NULLABLE,
+    ANY_TIMESTAMP_NOT_NULLABLE,
+    ANY_OBJECT_NOT_NULLABLE,
+
+    NOT_NULL;
```

Comment:
Would this be among the options? We already have ANY_NOT_NULLABLE.

Reply:
Removed NOT_NULL